libraries/Zigbee/src/ZigbeeHandlers.cpp

```diff
@@ -138,4 +140,25 @@ static esp_err_t zb_cmd_default_resp_handler(const esp_zb_zcl_cmd_default_resp_m
   return ESP_OK;
 }
 
+static esp_err_t zb_cmd_ias_zone_status_change_handler(const esp_zb_zcl_ias_zone_status_change_notification_message_t *message) {
+  if (!message) {
+    log_e("Empty message");
```

Comment:
you need to return here, or you will get exception further down when trying to access the structure

Reply:
It needs to be fixed in all handlers. Will do it in separate PR.

Reply:
Since this is new code, why not fix this one here and you can still add a PR to fix the rest @P-R-O-C-H-Y

Reply:
Sure, I was thinking to do it the same way you proposed.